Commands/Public/xkcd.js

```diff
@@ -0,0 +1,49 @@
+const fetch = require("chainfetch");
+
+module.exports = async ({ Constants: { Colors, Text, APIs, UserAgent } }, documents, msg) => {
+	const comicData = await fetch.get(APIs.XKCD(msg.suffix ? msg.suffix : undefined)).set({ "User-Agent": UserAgent, Accept: "application/json" }).onlyBody()
+		.catch(() => null);
+
+	const notFound = () => msg.send({
+		embed: {
+			color: Colors.SOFT_ERR,
+			description: msg.suffix ? `Doh! A XKCD comic with ID ${msg.suffix} wasn't found.` : `Doh! The current XKCD comic couldn't be fetched.`,
```

Comment:
Commenting this as a new review again :P
It's pronounced "ex kay cee dee", so it's `an`, not `a`. It's the sound that decides, not the letter. Unless you say it as a full word, in which case you should stop drinking.

Reply:
(Why do i keep blindly following vlad on english advice)